app/serializers/tag_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Class responsible for serializing tag data
+class TagSerializer < ActiveModel::Serializer
+  attributes :id, :name, :posts, :profiles
```

Comment:
As soon we have [this](https://github.com/iTechArt-RubyLab/social-network/pull/25/files#diff-33acbdd47076856064da679c623211f21bf6859f155a88a9df895e0a675e2a93R5) and it more likely will cause an infinite loop, we need to get rid of that.
TODO:
1) remove posts attribute
2) profiles attribute